packages/create-atomic/package.json

```
@@ -0,0 +1,33 @@
+{
+  "name": "@coveo/create-atomic",
+  "description": "Coveo Atomic Generator",
+  "version": "1.18.0",
+  "bin": {
+    "create-atomic": "./index.mjs"
+  },
+  "main": "index.mjs",
+  "scripts": {
+    "start": "plop",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "npm:bump:template": "npm --no-git-tag-version version",
```

Otherwise, linting is not run at the CI level.
```suggestion
    "npm:bump:template": "npm --no-git-tag-version version",
    "lint": "prettier --check . && eslint ."
```